analyzer/src/main/kotlin/AnalyzerResultBuilder.kt

```diff
@@ -61,23 +64,41 @@ class AnalyzerResultBuilder(private val curationProvider: PackageCurationProvide
             issues[existingProject.id] = projectIssues + issue
         } else {
             projects += projectAnalyzerResult.project
+            addPackages(projectAnalyzerResult.packages)
```

Comment:
The refactoring of `addResult` seems unrelated to the change and should have been done in a separate commit.

Reply:
Split to a separate commit.